container/target/target.tf

```diff
@@ -32,14 +32,6 @@ variable "tags" {
 # ------------------------------------------------------------------------------
 # Resources
 # ------------------------------------------------------------------------------
-locals {
-  default         = "${lookup(var.target, "health", "${var.target["protocol"]}:traffic-port/")}"
```

Comment:
This functionality is lost with the current changes. I.e., if nothing is specified we have a default.